blogposts/2021/release-3.27.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Sourcegraph 3.27 release"
+publishDate: 2021-04-20T10:00-07:00
+description: "Sourcegraph 3.27 is released."
+tags: [blog, release]
+slug: "release/3.27"
+published: true
+
+Sourcegraph 3.27 is now available! For this release, we focused on making Sourcegraph Batch Changes even better, added precise Code Intelligence for C++ and Java, and upgraded our minimum supported version of Postgres from 9.6 to 12.
+
+**Batch Changes Updates** 
+[Batch Changes](https://about.sourcegraph.com/batch-changes/) allows you to automate large-scale code changes with a single declarative file. With a simple UI, Batch Changes makes it easy to track and manage all of your changesets from checks to code reviews until each change is merged. 
+As part of this release you can now define Batch Changes rollout windows allowing you to better control when changes are rolled out, and we added the ability to mark changesets as “archived” when a new batch spec is created. 
+
+**Postgres update**
+We updated the minimum supported version of Postgres from 9.6 to 12. If you are maintaining an external database and your Postgres version is older than Postgres 12, you will need to update your database instance prior to upgrading from Sourcegraph 3.26 to 3.27. 
+[Read more about why we made this change](https://about.sourcegraph.com/blog/postgres-version-update/). 
+
+changelogItems:
+  # Added
+  - description: "`count:` now supports \"all\" as value. Queries with `count:all` will return up to 999999 results."
+    url: https://github.com/sourcegraph/sourcegraph/pull/19756
+    category:
+  - description: Credentials for Batch Changes are now validated when adding them.
+    url: https://github.com/sourcegraph/sourcegraph/pull/19602
+    category: Batch Changes
+  - description: "Batch Changes now ignore repositories that contain a `.batchignore` file. [#19877](https://github.com/sourcegraph/sourcegraph/pull/19877) and src-cli#509"
+    url: "https://github.com/sourcegraph/src-cli/pull/509"
+    category: Batch Changes
+  - description: "Side-by-side diff for commit visualization."
+    url: https://github.com/sourcegraph/sourcegraph/pull/19553
+    category:
+  - description: "The site configuration now supports defining batch change rollout windows, which can be used to slow or disable pushing changesets at particular times of day or days of the week. [#19796](https://github.com/sourcegraph/sourcegraph/pull/19796), [#19797](https://github.com/sourcegraph/sourcegraph/pull/19797), and #19951."
```

Comment:
```suggestion
  - description: "The site configuration now supports defining batch change rollout windows, which can be used to slow or disable pushing changesets at particular times of day or days of the week."
```

@content-hacker - should we include this in the body (in addition to the intro)?

Reply:
@abschu I think so. I see the intro as a highlights section and the changelog as the "full details" section.